theme/pxt.less

```diff
@@ -7,7 +7,10 @@
 @import 'tutorial';
 @import 'sidedoc';
 
+@import 'fieldeditors';
+
```

Comment:
I think this is meant to be in blockly-core.less so no need for this change.

Reply:
Good catch